template.yaml

```diff
@@ -536,6 +560,42 @@ Resources:
               - 's3:List*'
             Resource: '*'
 
+    #---- Buckets ----
+  LoaderBucket:
+    Type: AWS::S3::Bucket
+    Properties:
+      AccessControl: Private
+      BucketName: !Sub "${LoaderBucketName}-${AWS::AccountId}"
+      LifecycleConfiguration:
+        Rules:
```

Comment:
This may be too long an interval — the loader does anything other than read files